corehq/apps/hqwebapp/tasks.py

```diff
@@ -0,0 +1,57 @@
+from celery.task import task
+from celery.utils.log import get_task_logger
+from django.conf import settings
+from django.core.mail import send_mail
+from dimagi.utils.django.email import send_HTML_email
+
+logger = get_task_logger(__name__)
+
+
+@task(bind=True, default_retry_delay=10 * 60, max_retries=10)
```

Comment:
do you know offhand how the `retry` options interplay with `acks_late`? my gut would be that if celery dies  before the email gets sent then the tasks vanish. if that seems desired then is fine but worth acknowledging that's how we want it to work. IMO sending two emails may be better than sending 0 in that scenario, in which case if what I said is right then we should also set `acks_late=True`